src/quo2/components/wallet/network_link/view.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo2.components.wallet.network-link.view
+  (:require
+    [quo2.theme :as quo.theme]
+    [quo2.foundations.resources :as resources]
+    [react-native.core :as rn]
+    [react-native.fast-image :as fast-image]
+    [react-native.platform :as platform]))
+
+(def height
+  {:linear 10
+   :1x     66
+   :2x     122})
+
+(defn- view-internal
+  [{:keys [shape theme preview?]}]
+  ;; Updating image height on the fly (in preview screen) on Android does not work with fast-image
+  ;; so we are using rn/image for preview screen
+  (let [image-component (if (and platform/android? preview?) rn/image fast-image/fast-image)
```

Comment:
hmm I'm not sure if this is supposed to be an image - is it more of an svg type component? 🤔 
did you check the expected behaviour with the designers ?

Also I think the colors are not quite fixed but more of a linear gradient that depends on some configurations